pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
Please use the pattern we are using above (`if err := ...; err != nil { importDigest.Err = err; continue }`) for readability.

This definitely will be broken on v1, which does not have layer sizes.